Core/src/utils/Handler.cs

```diff
@@ -9,39 +9,49 @@
 
 namespace Core.utils
 {
-    public class FileHandler
+    public sealed class FileHandler
     {
 		/* singleton */
 		private FileHandler() { }
-		private static FileHandler singleton;
-		public static FileHandler getSingleton() {
-			if (singleton == null) singleton = new FileHandler();
-			return singleton;
+		private static readonly FileHandler singleton = new FileHandler();
+		public static FileHandler GetFileHandler {
+			get
+			{
+				return singleton;
+			}
 		}
 
 		public static void initialize() { /* TODO: impliment */ }
 
 		// System.IO.File.ReadAllText(file_path);
 		
 
-
-
 		/* implement I/O interface for file access.
 		 * sebe client request cache, downloaded respones cache
 		 * also log files -> application, sebe client response logging
 		 */
+
+		/* Copy a File */
+		public void CopyFile(string sourceFileName, string destinationFileName, bool overwrite) {
+
+			File.Copy(sourceFileName, destinationFileName, overwrite);
```

Comment:
`CopyFile(string sourceFileName, string destinationFileName, bool overwrite)` method might be unnecessary.  can use standard way `File.Copy()`

Reply:
If error is not handled here, this method is unnecessary, but if we were to handle file not found or rewrite errors here, than this method is necessary.